### PR TITLE
Add version check for JavaConsistentQuery

### PR DESCRIPTION
**Pull request by longquanzheng**
**What changed?**
Add version check for JavaConsistentQuery

**Why?**
Will merge after https://github.com/uber/cadence-java-client/pull/530 is merged

**How did you test it?**
No test needed

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: common/client/versionChecker.go [hunk 1]
```diff
@@ -56,6 +56,8 @@ const (
 	GoWorkerConsistentQueryVersion = "1.5.0"
 	// JavaWorkerRawHistoryQueryVersion indicates the minimum client version of the java worker which supports RawHistoryQuery
 	JavaWorkerRawHistoryQueryVersion = "1.3.0"
+	// JavaWorkerConsistentQueryVersion indicates the minimum client version of the java worker which supports RawHistoryQuery
```
Comment (vytautas-karpavicius) [inline common/client/versionChecker.go:59]: nit: please update the comment `RawHistoryQuery` -> `consistent query`

File: common/client/versionChecker.go [hunk 2]
```diff
@@ -56,6 +56,8 @@ const (
...
+	JavaWorkerConsistentQueryVersion = "1.4.0"
```
Comment (vytautas-karpavicius) [inline common/client/versionChecker.go:60]: `1.4.0` is this the release version? If so we should probably target `3.2.0`

Comment (vytautas-karpavicius) [reply common/client/versionChecker.go:60]: Ahh, never mind its a FEATURE_VERSION. We should probably bump it though?

File: common/client/versionChecker.go (hunk 1 above)
Comment (longquanzheng) [reply common/client/versionChecker.go]: Yeah fix it

File: common/client/versionChecker.go (hunk 2 above)
Comment (longquanzheng) [reply common/client/versionChecker.go]: Yep bumped it to 1.5

**Review by vytautas-karpavicius — APPROVED** [review]

**Pull request merged**